verify.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'enumerable'
+
+class List
+  @list = []
+  include Enumberable
+
+  def initialize(list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] As suggested by the previous reviewer, please implement `#each` method so that it yields all members of `@list` in succession. As a hint here, please refer to the pieces of code below 👇 

```ruby
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```